test/arrays/userAPI/arrayAPItest.chpl

```diff
@@ -0,0 +1,152 @@
+config param testError = 0;
+
+proc readArray(X) {
+  open("realValues.txt", iomode.r).reader().read(X);
```

Comment:
I think you should close() the channel and file here.  We plan to reference count these, but I don't think we currently do.